server/src/main/java/timely/api/model/Metric.java

```diff
@@ -34,10 +34,23 @@
 @WebSocket(operation = "put")
 public class Metric implements TcpRequest, HttpPostRequest, WebSocketRequest {
 
+    private static final ThreadLocal<ByteBuffer> WRITE_BUFFER = new ThreadLocal<ByteBuffer>() {
```

Comment:
There are multiple threads creating instances of the Metric class. I was just trying to reduce object creation here. I don't think a ByteBufferPool exists in the Java 8 JDK api. Creating one, using commons-pool for example, may incur the same overhead.